examples/e-commerce/src/theme.css

```diff
@@ -10,6 +10,7 @@ a[class^='ais-'] {
   text-decoration: none;
 }
 
+[class^='ais-'][class$='--disabled'],
 [class^='ais-'][class$='--disabled'] * {
```

Comment:
Do we need this?
I've tested the vanillia with only `[class^='ais-'][class$='--disabled'] { ... }` and it works fine.
And I think it's supposed to work without `*`.
What do you think?

Reply:
We need this line for widgets like RangeSlider which contain multiple components with several cursors. Ohterwise, the "not-allowed" cursor gets overridden.

I can add a comment to explain why in the theme.

Reply:
Ah I see. It'd be good to have that comment since this one is `theme.css`, but not `app.css`.